### PR TITLE
Move planner from STM32 to Raspberry Pi

### PR DESCRIPTION
**Pull request by ecourtois**
This first version implements only a very basic planner with a static path, no avoidance.

### CODE REVIEW

File: cogip/tools/monitor/socketiocontroller.py [hunk 1]
```diff
@@ -24,8 +23,10 @@ class SocketioController(QtCore.QObject):
             Qt signal emitted to log messages in UI console
         signal_new_menu:
             Qt signal emitted to load a new menu
-        signal_new_robot_pose:
-            Qt signal emitted on robot pose update
+        signal_new_robot_pose_current:
+            Qt signal emitted on robot pose current update
+        signal_new_robot_order:
+            Qt signal emitted on robot pose order update
```
Comment (gdoffe) [inline cogip/tools/monitor/socketiocontroller.py:29]: This commit does more than just "remove unused import".
Should be splitted.

File: cogip/models/__init__.py [hunk 2]
```diff
@@ -1,2 +1,3 @@
 from .models import CtrlModeEnum, DynObstacleList, DynObstacleRect, DynRoundObstacle, Vertex, Pose, RobotState  # noqa: F401
```
Comment (gdoffe) [inline cogip/models/__init__.py:1]: In https://github.com/cogip/simulation/pull/39/commits/cf3734eaf6aab69b8d3ca149f668e3da1a4016bb commit message:
s/This/These/

File: cogip/models/models.py [hunk 3]
```diff
@@ -119,6 +121,65 @@ class Speed(BaseModel):
     angle: float = 0.0
 
 
+class SpeedEnum(IntEnum):
+    """
+    Speed levels.
+    In mcu-firmware, the speeds (linear or angular) are float-point values,
```
Comment (gdoffe) [inline cogip/models/models.py:127]: s/or/and/

File: cogip/models/models.py [hunk 4]
```diff
@@ -119,6 +121,65 @@ class Speed(BaseModel):
...
+    but then can take only 3 values: low, normal and max speed. These values
```
Comment (gdoffe) [inline cogip/models/models.py:128]: s/then/they/ ?

File: cogip/models/models.py [hunk 5]
```diff
@@ -119,6 +121,65 @@ class Speed(BaseModel):
...
+    MAX = 2
+
+
+class PathPose(BaseModel):
+    """
+        Class representing a position in a path.
+
+        Attributes:
+            x: X coordinate
+            y: Y coodinate
```
Comment (gdoffe) [inline cogip/models/models.py:148]: coordinate

File: cogip/tools/copilot/sio_events.py [hunk 6]
```diff
@@ -78,31 +77,13 @@ async def on_cmd(self, sid, data):
         response.cmd, _, response.desc = data.partition(" ")
         await self._copilot.send_serial_message(server.command_uuid, response)
 
-    async def on_wizard(self, sid, data):
+    async def on_planner_cmd(self, sid, data):
```
Comment (gdoffe) [inline cogip/tools/copilot/sio_events.py:81]: This is not a removal. Should be in previous commit ?

File: cogip/tools/planner/__init__.py [hunk 7]
```diff
@@ -0,0 +1,7 @@
+import logging
+
+logging.basicConfig(
+    level=logging.INFO,  # logging.DEBUG
```
Comment (gdoffe) [inline cogip/tools/planner/__init__.py:4]: dead code in comment

**Review by gdoffe — CHANGES_REQUESTED** [review]
Good job @Starbeuck @ecourtois !

Mainly typo and a commit to spit.

File: cogip/tools/monitor/socketiocontroller.py (hunk 1 above)
Comment (ecourtois) [reply cogip/tools/monitor/socketiocontroller.py]: Done.

**Review by gdoffe — APPROVED** [review]

**Pull request merged**